op-node/rollup/driver/step.go

```diff
@@ -198,9 +198,14 @@ func (d *outputImpl) insertEpoch(ctx context.Context, l2Head eth.L2BlockRef, l2S
 	if err != nil {
 		return l2Head, l2SafeHead, false, fmt.Errorf("failed to fetch transactions from %s: %v", l1Input, err)
 	}
-	batches, err := derive.BatchesFromEVMTransactions(&d.Config, transactions)
-	if err != nil {
-		return l2Head, l2SafeHead, false, fmt.Errorf("failed to fetch create batches from transactions: %w", err)
+	batches, errs := derive.BatchesFromEVMTransactions(&d.Config, transactions)
```

Comment:
This receives all transactions in a block right? If so, then an invalid `tx.to` isn't an error